api/orchestrator/orchestrator.proto

```diff
@@ -778,6 +786,13 @@ message TargetOfEvaluation {
   ];
 }
 
+message GetAssessmentResultRequest {
+  string id = 1
+      [ (validate.rules).string.uuid = true ];
+  string cloud_service_id = 2
```

Comment:
svc.authz.CheckAccess requires a cloud_service_id.

In case of no cloud_serivce_id, the following error occurs:
`Cannot use 'req' (type *orchestrator.GetAssessmentResultRequest) as the type api.CloudServiceRequest Type does not implement 'api.CloudServiceRequest' as some methods are missing: GetCloudServiceId() string`

What's the problem of having the cloud_service_id?

Reply:
The CheckAccess method requires a cloud_service_id. 
As far as I understood this comment (https://github.com/clouditor/clouditor/pull/1026#discussion_r1147925621) correctly, authorization should be added to this get-request. 

Other get-requests whose request parameter type has no cloud_service_id, do not have authorization checks
(i.e. GetMetricImplementation())

Reply:
In the case of `GetMetricImplementation`, the metric implementation is independent from a cloud service. But if there are other requests whose result is dependent on a cloud service, feel free to create an issue :)

Reply:
> The CheckAccess method requires a cloud_service_id. As far as I understood this comment ([#1026 (comment)](https://github.com/clouditor/clouditor/pull/1026#discussion_r1147925621)) correctly, authorization should be added to this get-request.
> 
> Other get-requests whose request parameter type has no cloud_service_id, do not have authorization checks (i.e. GetMetricImplementation())

Yes, but the problem is, 
a) nobody is filling the `cloud_service_id`, because the http pattern for request only contains the id (`get : "/v1/orchestrator/assessment_results/{id}"`). So the cloud service ID is always empty and the check would probably always fail.
b) you are only including the ID in the query to the database. So even if first check would succeed, you could retrieve any result by their ID, even if it's associated to another cloud service.

I discussed possible solutions in the other comment below

Reply:
Roger that